index.user.js

```diff
@@ -117,7 +121,15 @@ async function downloadAudio() {
     }
     const aNode = document.getElementById('IdDownloadAudio');
     const playInfo = getMediaInfo();
-    const responsePayload = await fetch(`https://foamzou.com/tools/bilibili/fetchAudio.php?url=${encodeURIComponent(playInfo.audioUrl)}&name=${playInfo.name}&vid=${playInfo.vid}`);
+    const startTime = document.getElementById('audioStartTime').value.trim();
+    const endTime = document.getElementById('audioEndTime').value.trim();
+    
+    const url = `https://foamzou.com/tools/bilibili/fetchAudio.php?url=${encodeURIComponent(playInfo.audioUrl)}&name=${playInfo.name}&vid=${playInfo.vid}`;
+    const fullUrl = startTime || endTime ? 
+        `${url}&start_time=${encodeURIComponent(startTime)}&end_time=${encodeURIComponent(endTime)}` : 
```

Comment:
This assumes the backend service will support the 2 new parameters. Let me know if I should just make impacts to the other 2 comamnds, not this download action.

Reply:
removed.